app/src/main/java/com/example/pokemonapp/ui/PokemonMainActivityDaggerHilt.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.pokemonapp.ui
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class PokemonMainActivityDaggerHilt :Application() {
```

Comment:
What do you think to rename this class to PokemonApplication

Reply:
done